src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = days * 40;
+
+  if (days >= 7) {
+    return cost - 50;
+  } else if (days >= 3 && days < 7) {
```

Comment:
you already checked that days < 7
```suggestion
  } else if (days >= 3) {
```